library/api.inc

```diff
@@ -1,6 +1,6 @@
 <?php
 //our api for 3rd party developers
-include_once("/../interface/globals.php");
+include_once("../../interface/globals.php");
```

Comment:
![image](https://user-images.githubusercontent.com/6632800/34645518-bdc1e8f0-f347-11e7-9a07-ecb81783e886.png)

Look, something's wrong... try to remove the first ../ and test it

Reply:
require_once '../interface/globals.php';
Up one level, down one level.

Reply:
Not working :/